src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -154,7 +216,8 @@ def main():
 
     # create a map containing midlines for each clump
     gs.verbose(_("Finding median lines"))
-    categorymap_thin = "{}".format(categorymap) + "_thin"
+    categorymap_thin = f"{categorymap}_thin"
+    CLEAN_LAY.append(categorymap_thin)
```

Comment:
Just FYI, another way to do it is with the atexit.register method:
https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md#temporary-maps